### ISSUE TITLE
RMS-Lint alert when declare same constants  in conditions

### ISSUE THREAD

**Issue by fntsrlike** [mangudai/vscode#1]
## Info
- VS Code version:  `1.16.1`
- Extension version: `0.2.9`

## Description
When I declare constants in conditions, it will not be duplicate declaration, but RMS-Lint still alert.

### Except
It should only alert when code really duplicate.

### Actually
It alert when I declare same constant name, but it will not duplicate because there has conditions.

#### Screenshot
![2017-09-24 7 54 46](https://user-images.githubusercontent.com/2680724/30778056-d173ef34-a0fd-11e7-8cd4-3ddf580833bf.png)

#### Code
```
if TINY_MAP
	#const EXTRA_FORAGE_GROUPS 1
	#const EXTRA_FORAGE_OBJECTS 1
elseif SMALL_MAP
	#const EXTRA_FORAGE_GROUPS 2
	#const EXTRA_FORAGE_OBJECTS 2
elseif MEDIUM_MAP
	#const EXTRA_FORAGE_GROUPS 3
	#const EXTRA_FORAGE_OBJECTS 3
elseif LARGE_MAP
	#const EXTRA_FORAGE_GROUPS 4
	#const EXTRA_FORAGE_OBJECTS 4
elseif HUGE_MAP
	#const EXTRA_FORAGE_GROUPS 5
	#const EXTRA_FORAGE_OBJECTS 5
elseif GIGANTIC_MAP
	#const EXTRA_FORAGE_GROUPS 6
	#const EXTRA_FORAGE_OBJECTS 6
elseif LUDIKRIS_MAP
	#const EXTRA_FORAGE_GROUPS 7
	#const EXTRA_FORAGE_OBJECTS 7
else
	#const EXTRA_FORAGE_GROUPS 0
	#const EXTRA_FORAGE_OBJECTS 0
endif
```

**Comment by deltaidea** [comment]
Thank you so much for these detailed reports!
I'm aware of this one and plan to fix it once I have time.

Have a nice day! 😄

**Comment by deltaidea** [comment]
Fixed in 0.3.1

![image](https://user-images.githubusercontent.com/4950036/42415531-36acda62-825a-11e8-86cb-674a39e0a56f.png)
